primer.md

```diff
@@ -289,6 +289,28 @@ context, for the purposes of this CloudEvent attribute those meanings are not
 relevant and therefore using `id` for some other purpose beyond uniqueness
 checking is out of scope of the specification and not recommended.
 
+### data
+
+The `data` attribute of a CloudEvent may carry the event data itself. Event
```

Comment:
Actually, `data` is not regarded as attribute in v1.0.  If I recall it correctly, that was changed between 0.3 and 1.0, due to a lot of discussions around the CloudEvents type system and the attribute `datacontentencoding` that existed in 0.3.

Reply:
https://github.com/cloudevents/spec/blob/master/spec.md#event-data
I wonder if we should just drop the first sentence entirely - doesn't seem to add much.

Reply:
>  I wonder if we should just drop the first sentence entirely - doesn't seem to add much.

Done